src/models/sunburstChart.js

```diff
@@ -126,6 +127,10 @@ nv.models.sunburstChart = function() {
             color = _;
             sunburst.color(color);
         }},
+        groupColorByParent: {get: function(){return groupColorByParent;}, set: function(_){
```

Comment:
You don't need to add this here, it inherits the options from sunburst.js

See this line below:
    nv.utils.inheritOptions(chart, sunburst);